src/GPRS.cpp

```diff
@@ -320,6 +320,10 @@ IPAddress GPRS::getIPAddress()
   return IPAddress(0, 0, 0, 0);
 }
 
+unsigned long GPRS::timeout() {
```

Comment:
Please use the brace style established by the existing code:
```suggestion
unsigned long GPRS::timeout()
{
```